<?xml version="1.0"?>
<launch>
    <arg name="sim" default="false"/>

    <rosparam file="$(find bitbots_ros_control)/config/wolfgang.yaml" command="load" />

    <node pkg="bitbots_ros_control" type="dynamixel_controller_manager_node" name="ros_control" output="screen"/> 

    <rosparam file="$(find bitbots_ros_control)/config/controller.yaml" command="load" />

    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
          output="screen" args="--timeout 20 joint_state_controller imu_sensor_controller DynamixelController ">
    </node>

    <node pkg="diagnostic_aggregator" type="aggregator_node" name="diagnostic_aggregator" >    
        <rosparam command="load" file="$(find bitbots_ros_control)/config/analyzers.yaml" />
    </node>

    <include file="$(find bitbots_ros_control)/launch/imu.launch">
    </include>

</launch>